src/graphics/spritebatch.rs

```diff
@@ -82,6 +82,20 @@ impl SpriteBatch {
         }
     }
 
+    /// Returns a reference to the sprites.
+    pub fn get_sprites(&self) -> &[DrawParam] {
+        &self.sprites
+    }
+
+    /// Returns a mutable reference to the sprites.
+    ///
+    /// Please note that manually altering items in this slice
```

Comment:
Is this note really necessary? It automatically gets flushed in the draw() call and clear() doesn't say anything about it either.

Reply:
I copy pasted this docstring from MeshBatch, so I figured might as well do it for consistency there. I can remove it if it's unnecessary.

Reply:
Manually flushing the spritebatch is really not necessary, as spritebatch currently does not feature a dirty bit like MeshBatch. Then again, we recently thought about adding one, so it might be worth to keep this in, just so that people don't stumble over this later.

I'm a bit torn. The best thing might be to change the comment, so that it says that manually flushing the spritebatch is currently not necessary, but might become necessary in the foreseeable future, due to a dirty bit that might be added to spritebatch internally.